src/server/migrations/20200619230308_game_timer_field.js

```diff
@@ -0,0 +1,10 @@
+exports.up = function (knex) {
+    return knex.schema.table('game_scores', (table) => {
+      table.integer('game_timer');
+      
```

Comment:
Hi @myshuker. I think that instead of integer this should be a format of HH:MM:SS. 
I was thinking how are we gonna calculate this time too.

Reply:
And were you able to roll back the migration as well?

Reply:
thanks Lucha.
The error that got in building is because I dont push (game_scores) table with my PR coz this table not me and its already in develop branch....

Reply:
and regarding the format of the time I was thinking about that but I dont know if format('YYYY-MM-DD') sould I use or another... what you think?.

Reply:
> And were you able to roll back the migration as well?
you are right ,.,..
I correst it now